### ISSUE TITLE
About yolovn

### ISSUE THREAD

**Issue by lucasjinreal** [lucasjinreal/yolovn#1]
Some people trends to criticize others should not naming next generation yolo, but **themself use next generation yolo name so natural like it's their own property**.

I suggest these people better to do their own jobs and do not waste time on double standard. 

So here is all I will do: yolovn comes out.

You can choose yolov8 yolov9 yolov7 anything you want, but **do not annoying our repo and waste both you and my time on arguing about naming**.

**Comment by JiangWeiHn** [comment]
you are right!! 为你点赞

**Comment by HuynhLongTGG** [comment]
yolovn makes me think yolo VietNam =))

**Comment by lucasjinreal** [comment]
@HuynhLongTGG  also can be: yolo V fuck Not need that number. These wuk yolov7 release their paper and name half year behind us, and claims that naming it's their property which is very pathetic and disgusting. Some admirers even has no brain to sneer us didn't even have a look our repo stars more than them, our release date half year earlier than them.

**Comment by HuynhLongTGG** [comment]
hahaha, Ignore them and let's just do what we think is right. I'm implementing a license plate recognition system for 600 cameras in Vietnam, and still learning every day from you and others. We only need knowledge, not fame. i want to optimize the current infrastructure and improve the algorithm, if possible can you let me learn more from you

**Comment by lucasjinreal** [comment]
@HuynhLongTGG Yes, that's what I want: **share my experiences and build a powerful tool for everyone, rather than using some Already have techniques and release a paper than claim that THIS IS MY PROPERTY**.  I would like do more solid work,  try more experiment and put these **steps** to everyone, rather than sharing a beautiful AP number to everyone without any intermediate results, failing experiments and experiences, this is much more important than just release a beautiful number. I hope we can grow together along with our community.

**Comment by HuynhLongTGG** [comment]
@jinfagang  thanks u very much , my telegram is @longhtgg96, i just dm you. I have a lot of labeled data about Vehicle , but it's really hard to optimize it when implementing a real project. I also hope that I will find a suitable solution to share with the community

**Comment by lucasjinreal** [comment]
For anyone who get a chance to know our project. I am very pleased to let everyone know **how a Chinese people can be in such a rube manner on a simple thing**, I feel so shame as a Chinese about this guy's manner:

<img width="476" alt="image" src="https://user-images.githubusercontent.com/21303438/178094452-60eb8954-62d7-4a68-a75d-d3a858adf6f7.png">
<img width="760" alt="image" src="https://user-images.githubusercontent.com/21303438/178094541-0ac824c5-780e-42dc-bc51-8214d0e6160e.png">

**Comment by PLester** [comment]
学会接受这个世界的参差，世界也许会变得可爱一些，把这个项目好好做下去。

**Comment by lucasjinreal** [comment]
人与人的认知差距是很大的，不过这并不妨碍我们继续在未知的道路上探索，同行的人可以一起学到很多知识，只知道跟着别人犬吠的人终究不过只 人云亦云的 犬吠生物
